app/scripts/controllers/incoming-transactions.js

```diff
@@ -61,33 +61,33 @@ class IncomingTransactionsController {
     }, opts.initState)
     this.store = new ObservableStore(initState)
 
-    this.preferencesController.store.subscribe(pairwise((prevState, currState) => {
```

Comment:
This consolidation of the subscription method was the first step toward implementing a `reset` function for this controller, which was never finished. I should probably remove these changes until they become useful.